web/src/app/profile/approved-researcher/components/AgreementText.tsx

```diff
@@ -1,7 +1,8 @@
 "use client";
 
+import "./AgreementText.css";
+import Markdown from "react-markdown";
 import { useEffect, useState } from "react";
-
 import { getAgreementsApprovedResearcher } from "@/openapi";
```

Comment:
I wonder if we want to use an eslint rule to standardise the import orders? Not for this PR though, I might make a ticket for it and we can discuss.